en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -472,4 +473,75 @@ To create a For Each Loop:
 
 You can add logic and actions in different patterns, depending on what your flow needs to do.
 
+== Designate Error Handling 
+
+When executing a flow, you can watch for errors in connection or logic in specific steps of your flow and then designate logic to handle the errors that may occur. You can also designate steps to complete additional actions or to send notifications in the event of an error. 
```

Comment:
```suggestion
When executing a flow, you can watch for connection or logic errors in specific steps of your flow and then designate logic to handle the errors. You can also designate steps to complete additional actions or to send notifications if an error occurs. 
```

Reply:
designate -> specify? (two instances of designate)

Reply:
Changed!